RoadMap_SwiftUI/RoadMap_SwiftUI/View/FilterView.swift

```diff
@@ -0,0 +1,159 @@
+            pricingStackView
+            Text("\(Constants.Texts.currency)\(viewModel.price.formatted())")
+            colorsStackView
+            colorGridView
+            Spacer()
+        }
+    }
+
+    // MARK: - Private properties
+    private let furnitures = [
+        Constants.Images.bedDouble,
+        Constants.Images.sofaSystem,
+        Constants.Images.chairLounge,
+        Constants.Images.systemChair,
+        Constants.Images.cabinet,
+        Constants.Images.systemTable
+    ]
+
+    private let colors: [Color] = [
+        .yellow,
+        .purple,
+        .orange,
+        .gray,
+        .red,
+        .blue,
+        .brown,
+        .cyan,
+        .indigo,
+        .mint,
+        .teal,
+        .primary,
+        .secondary,
+        .black,
+        .bloody,
+        .blueSky,
+        .yellowest,
+        .pinky,
+        .deepPurple,
+        .darkForest
+    ]
+
+    private let rows: [GridItem] = [
+        .init(.flexible())
+    ]
+    private let colorGrid: [GridItem] = [
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible()),
+        .init(.flexible())
+    ]
+    
+    @StateObject private var viewModel = FilterViewModel()
+    
+    private var navigationTitleTextView: some View {
+        Text("Filters")
+            .fontWeight(.bold)
+            .foregroundColor(.white)
+            .font(.largeTitle)
+            .padding(.bottom, 15)
+    }
+    
+    private var categoryStackView: some View {
+        HStack {
+            Text("Category")
+                .font(.title)
+                .fontWeight(.bold)
+            Spacer()
+            Text("More")
+                .font(.title)
+                .fontWeight(.bold)
+                .foregroundColor(.gray)
+            Image(systemName: "chevron.right")
+                .padding(.top, 1)
+        }
+    }
+    
+    private var furnituresStackView: some View {
+        ScrollView(.horizontal, showsIndicators: false) {
+            LazyHGrid(rows: rows) {
+                ForEach(furnitures, id: \.self) { product in
+                    ZStack(alignment: .center) {
+                        RoundedRectangle(cornerRadius: 0)
+                            .fill(Color.white)
+                            .shadow(radius: 3)
+                            .frame(width: 120, height: 75)
+                        Image(systemName: product)
+                            .resizable()
+                            .frame(width: 50, height: 50)
+                            .foregroundColor(.yellow)
+                    }
+                }
+            }
+        }
+    }
+    
+    private var pricingStackView: some View {
+        VStack {
+            HStack {
+                Text("Pricing")
+                    .font(.title)
+                    .fontWeight(.bold)
+                    .padding(.trailing, 250)
+            }
+            Slider(value: $viewModel.price, in: 100...5000, step: 500)
```

Comment:
```suggestion
            Slider(value: $viewModel.price, in: 100...5000, step: 500)
```
Думаю значения интервала и шага тоже нужно вынести в константы